pkgs/applications/virtualization/xhyve/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, lib, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "xhyve-${version}";
+  version = "0.1.0";
+
+  src = fetchurl {
+    url = "https://github.com/mist64/xhyve/archive/v${version}.tar.gz";
+    sha256 = "0nbb9zy4iqmdz2dpyvcl1ynimrrpyd6f6cq8y2p78n1lmgqhrgkm";
+  };
+
+  buildPhase = ''
+    make CFLAGS="-Wno-pedantic -Wno-shift-sign-overflow"
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp build/xhyve $out/bin
+  '';
+
+  checkPhase = ''
+    xhyve -h
```

Comment:
Not sure if it makes sense to call the program there. Usually this phase is used to execute (unit) tests.